typos.toml

```diff
@@ -0,0 +1,2 @@
+[type.rust]
+extend-ignore-identifiers-re = ["exten"]
```

Comment:
question: where does `exten` affect us?

Me, not remembering the source code (and not feeling like looking for it) but I'm guessing you've suffered somewhere that this affects.

Reply:
Correct

https://github.com/seffradev/asterisk-rs/blob/3822ec3db9a2bfa20573809490a3ff6981f22f48/arirs/src/channel.rs#L206-L208